static/js/topic_list.js

```diff
@@ -159,7 +178,41 @@ export class TopicListWidget {
         return this.my_stream_id;
     }
 
+    update_topic_search_text() {
+        const input = this.parent_elem.find("#filter-topic-input");
+        if (input.length) {
+            this.topic_search_text = input.val();
+        } else {
+            this.topic_search_text = "";
+        }
+    }
+
+    update_topic_search_input() {
+        const input = this.parent_elem.find("#filter-topic-input");
+        if (input.length) {
+            // update search text.
```

Comment:
This comment I think should be something like "Restore saved topic search text after the element was rerendered."

Reply:
Updated this comment!